pycobertura/reporters.py

```diff
@@ -234,6 +244,38 @@ def generate(self):
         return tabulate(lines, headers=headers)
 
 
+class CsvReporterDelta(DeltaReporter):
+    not_available = "-"
```

Comment:
Shouldn't this just be `not_available = ""` for csv? An empty string would represent a null value.

Reply:
true, done in https://github.com/aconrad/pycobertura/pull/130/commits/0c9701ad3a64ed8668910af42a59eb3a30b27592 :)